grisera_api/activity/activity_model.py

```diff
@@ -0,0 +1,51 @@
+from typing import List
+from pydantic import BaseModel
+from typing import Optional, Any
+from enum import Enum
+from property.property_model import PropertyIn
+
+
+class Type(str, Enum):
+    """
+    The type of activity
+
+    Attributes:
+        individual (str): Individual activity
+        two_people (str): Two people activity
+        group (str): Group activity
+    """
+    individual = "individual"
+    two_people = "two-people"
```

Comment:
Should we use `-` in enums' strings? @RioluM, in `observable_information_model.py` we've got spaces. I think we should be consistent. WDYT?

Reply:
Oh, indeed it's used as a one word. In that case i would leave it with -. What do you think @mrwrob ?